int64 left_speed
int64 right_speed
---
int8 result
